src/tribol/common/Parameters.hpp

```diff
@@ -493,6 +493,7 @@ struct parameters_t
   int vis_cycle_incr;            ///! Frequency for visualizations dumps
   VisType vis_type;              ///! Type of interface physics visualization output
   std::string output_directory;  ///! Output directory for visualization dumps
+  bool enable_timestep_vote;     ///! True if host-code desires the timestep vote to be calculated and returned
```

Comment:
could the host code instead provide the alpha multiplier for the timestep vote? (0 for disabled)

Reply:
The bool is fine... I was actually thinking about the timestep_pen_frac, which looks like it is already exposed.  I think alpha should be set to 1.0 rather than 0.75.